src/cephadm/cephadm.py

```diff
@@ -2939,10 +2941,13 @@ def apply_deploy_config_to_ctx(
     facade = ArgumentFacade()
     _add_deploy_parser_args(facade)
     facade.apply(ctx)
-    for key, value in config_data.get('params', {}).items():
+    params = config_data.get('params', {})
+    for key, value in params.items():
```

Comment:
Is this change functional? or just improvement? ... seems like the second case so unless previous code was really hard to read I'd recommend avoiding this kind of changes.